feedback.md

```
@@ -0,0 +1,110 @@
+In lib/eve_dmv/analytics/battle_detector.ex around lines 22-23 and also at lines
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)